django/db/backends/sqlite3/base.py

```diff
@@ -169,6 +169,25 @@ def get_new_connection(self, conn_params):
         conn.create_function("regexp", 2, _sqlite_regexp)
         conn.create_function("django_format_dtdelta", 3, _sqlite_format_dtdelta)
         conn.create_function("django_power", 2, _sqlite_power)
+        conn.create_function('ACOS', 1, math.acos)
+        conn.create_function('ASIN', 1, math.asin)
+        conn.create_function('ATAN', 1, math.atan)
+        conn.create_function('ATAN2', 2, lambda x, y: math.atan2(x, y))
```

Comment:
Sorry - my fault - don't need to wrap this one in a lambda.

Reply:
@JunyiJ - I think you missed this one.

Reply:
Oops. Right. I have updated it. Thanks.